zulipterminal/ui_tools/views.py

```diff
@@ -1098,7 +1098,8 @@ def timestamp_to_local_datetime(timestamp: int) -> str:
         member_keys = ', '.join(map(repr, keys_for_command('STREAM_MEMBERS')))
         stream_email = stream['email_address']
 
-        weekly_msg_count = stream['stream_weekly_traffic']
+        weekly_msg_count = ('N/A' if stream['stream_weekly_traffic'] is None
+                            else str(stream['stream_weekly_traffic']))
```

Comment:
You're referring to a commit, not a PR, in this commit text.

If `None` is always due to it being just-created, it would be useful to indicate that, and expand N/A to something clearer in any case.

I'd be happy merging this commit sooner, but the tests in the first commit are dependencies of all the others.

Reply:
Adding this commit in the same PR has led to this. I asked in #zulip-terminal about adding this fix in this PR or another PR, and we decided to go with the former. If you want, I can add this to the PR I mentioned few comments above. This should segregate this fix from the class parameters I declared in the prior commit?